package.json

```diff
@@ -47,10 +47,5 @@
     ],
     "*.{jsx,js}": "eslint --cache --fix",
     "*.{js,css,md}": "prettier --write"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
```

Comment:
- Updated upload button
- Tab group
   ---adjusted bg to grey-shd7 instead of shd6, note that bg colour for tab is different while selected than in neutral state in design, while parent styling on "check" can not be applied via CSS or tailwind, and we are not using JS, the same style on active as in design would appear on hover for now.
  ---adjusted font size and padding
---radio buttons styled.
-adjusted input box.